src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/ImprisonmentStatusHistoryResourceImplTest.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.justice.hmpps.prison.api.resource.impl;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+
+import java.util.List;
+
+public class ImprisonmentStatusHistoryResourceImplTest extends ResourceTest {
+    @Test
+    public void courtDateResults() {
+        final var requestEntity = createHttpEntityWithBearerAuthorisation("ITAG_USER", List.of("ROLE_VIEW_PRISON_DATA"), null);
+
+        final var responseEntity = testRestTemplate
+            .exchange(
+                "/api/inmate-status-history/A1180HL",
```

Comment:
Is it okay to use existing test data here?

Reply:
yes fine, all new files should be in kotlin though